src/porepy/viz/exporter.py

```diff
@@ -863,6 +927,7 @@ def write_pvd(
     def _sort_and_unify_data(
         self,
         data=None,
+        data_type: str = "num_cells",
```

Comment:
Instead of `str`, I suggest use of `Literal`, indicating the two possible choices.

I do not like the discrepancy in the naming. `num_cells` sounds like a number and not a data type. I would try to use a different name for `data_type`. What about `num_entities` or so? Maybe you have a better idea than me.

Reply:
No, I'm terrible in namings (@keileg knows). I change it to num_entities.